yarn.lock

```diff
@@ -1498,7 +1498,7 @@
   dependencies:
     "@jridgewell/trace-mapping" "0.3.9"
 
-"@ctrl/tinycolor@3.3.4":
+"@ctrl/tinycolor@3.3.4", "@ctrl/tinycolor@^3.3.4":
```

Comment:
Is there something still requesting the hardcoded 3.3.4 version?

Reply:
There is, the `package.json` at the root, but that should use strict versioning. I'll share with you the thread I had with Martin separately.